rs/nns/governance/tests/governance.rs

```diff
@@ -573,6 +573,12 @@ fn test_single_neuron_proposal_new() {
                         ),
                     ),
             ]),
+            NNSStateChange::LatestGcNumProposals(
```

Comment:
Nit. I'd prefer to use camel case as follows: 

```suggestion
            NnsStateChange::LatestGcNumProposals(
```